packages/homepage/content/articles/how-to-code-ipad-ide/index.md

```diff
@@ -4,153 +4,254 @@ slug: how-to-code-your-app-using-the-codesandbox-ipad-ide
 authors: ['Carlos Vidal']
 photo: https://avatars.githubusercontent.com/u/1814571?v=4
 title: How to code your app using the CodeSandbox iPad IDE
-description: In this in-depth tutorial, we showcase the developer experience of the CodeSandbox iPad IDE. 
+description:
+  In this in-depth tutorial, we showcase the developer experience of the
+  CodeSandbox iPad IDE.
```

Comment:
Not sure where this file change comes from, I'll merge `master` into this branch first.

Reply:
Just formatting.